doc/audio.md

```diff
@@ -1,3 +1,9 @@
+<!-- {ignore} -->
+## ⚠️ Warning:
+This is a source file for the docs site and may not be up to date with the latest version.
+Prefer to read it on [The official docs site](https://flame-engine.org/docs)
+<!-- {/ignore} -->
+
```

Comment:
yeah I dont think we should add that to every file

Reply:
I agree, only on the README is enough